### PR TITLE
Add new announcement

### PR DESCRIPTION
**Pull request by NaughtyChas**
Added announcement for QF

We have to add another announcement for pick'em. Someone help me finish this plz as I have goucaode lecture this afternoon.

### CODE REVIEW

File: blog/2024.9.9-QF.md [hunk 1]
```diff
@@ -9,6 +9,20 @@ tags: [公告]
 
 <!-- truncate -->
 
+:::info 竞猜
+
+随着 Qualifier 的到来，OFFC24 Cirno 的竞猜也已启动！所有人均可参加，竞猜正确数目第一者可获得**一个月的 osu!supporter**！（若有并列，则选取投稿时间前三名）
+
+欢迎各位在 <Highlight color="#198dff">**[这里](https://challonge.com/zh_CN/tournaments/15088528/predictions/new)**</Highlight> 大胆填写对于赛事结果的预测与想象！
+
+您无需登录，只需要按网页的指示，将队伍拖入每轮比赛的预测结果空位中，然后在下方选择预测获胜方即可。
```
Comment (NaughtyChas) [inline blog/2024.9.9-QF.md:18]: Maybe we have to add more information on the format of the pick'em title. Users must include their osu! ID in the title.

File: blog/2024.9.9-QF.md [hunk 2]
```diff
@@ -9,6 +9,20 @@ tags: [公告]
 
 <!-- truncate -->
 
+:::info 竞猜
+
+随着 Qualifier 的到来，OFFC24 Cirno 的竞猜也已启动！所有人均可参加，竞猜正确数目第一者可获得**一个月的 osu!supporter**！（若有并列，则选取投稿时间前三名）
```
Comment (NaughtyChas) [inline blog/2024.9.9-QF.md:14]: Have we decided to change our tournament title to ˋOFFC24ˋ yet?

Comment (CloneWith) [reply blog/2024.9.9-QF.md:14]: Not yet? Don't know what they think on this.
Can change this to *OFFC 2024* if possible.

File: blog/2024.9.9-QF.md (hunk 1 above)
Comment (CloneWith) [reply blog/2024.9.9-QF.md:18]: Below this line already has something about that.
Should I clarify it with more detail, or move them here?

File: blog/2024.9.9-QF.md (hunk 2 above)
Comment (NaughtyChas) [reply blog/2024.9.9-QF.md:14]: On my side OFFC cirno is fine. We are not responsible for tournament name clash, in fact, it is not a big deal.

**Comment by NaughtyChas** [comment]
Todo: Add board and mappool information to this announcement

File: blog/2024.9.9-QF.md (hunk 1 above)
Comment (NaughtyChas) [reply blog/2024.9.9-QF.md:18]: Resolved.

**Review by CloneWith — APPROVED** [review]
Just 've fix some links and team names.

Now we could push this 😋 out

**Pull request merged**